src/coguard_cli/image_check/__init__.py

```diff
@@ -73,7 +78,7 @@ def find_configuration_files_and_collect(
         if len(discovered_config_files) > 0:
             collected_service_results_dicts[finder_instance.get_service_name()] = \
                 (finder_instance.is_cluster_service(), discovered_config_files)
-    dockerfile_entry = extract_docker_file_and_store(image_name)
+    dockerfile_entry = extract_docker_file_and_store(image_name) if not is_container_scan else None
```

Comment:
We might want to rethink this, as we might want to also scan the associated Dockerfile as well...

Reply:
revisited, and we included it now.